frontend/src/layout/navigation/TopNavigation.tsx

```diff
@@ -51,6 +54,53 @@ export function WhoAmI({ user }: { user: UserType }): JSX.Element {
     )
 }
 
+function ProjectRow({ team }: { team: TeamBasicType }): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const { updateCurrentTeam } = useActions(userLogic)
+    const { push } = router.actions
```

Comment:
Nit: This could also be a `useActions(router)` for consistency. Makes no real difference, as `push` is an action and `router` is always mounted, but still nit nit nit.